src/compute/src/server.rs

```diff
@@ -570,10 +570,19 @@ impl<'w, A: Allocate + 'static> Worker<'w, A> {
                                     ?uncompacted,
                                     ?subscribe_free,
                                     ?dependencies_retained,
-                                    old_as_of = ?old_dataflow.as_of,
-                                    new_as_of = ?as_of,
```

Comment:
I'd leave the `as_of` printing here, too, because it might be difficult to read it from the next log line if the dataflow plan is too big.

Reply:
Fair enough! I was thinking we could get the as-ofs from the "creating dataflow" logs so logging them again here is redundant. But the log for the old dataflow might be a couple days old so it might be hard to find it.